CONTRIBUTING.md

```diff
@@ -1,7 +1,56 @@
 # Contributing
 
+## Overview
 This project welcomes contributions and suggestions. Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit https://cla.microsoft.com.
 
 When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repositories using our CLA.
 
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+
```

Comment:
This should probably also include the getting started dev environment for codespaces, imo this document is the right place for that :)

Reply:
Yep I referenced that in the PR description. Just waiting on your docs PR to be merged first.

Reply:
gotcha!

Reply:
Longer term there should be a separate developer's guide (can be referenced from CONTRIBUTING.md), but for now it would be ok to just include it here.